lib/count_words.rb

```diff
@@ -0,0 +1,62 @@
+class CountWords
+
+    # parse text
+    def run(filename)
+
+        # read the text file
```

Comment:
i like the pipelining. Also see Mike Gee's comment on sticking some of this stuff into the constructor.